### DISCUSSION TITLE
Allow Navigating to a specific tab

### DISCUSSION THREAD

**Discussion by cloudbr34k84** [kinghat/tabbed-card#57] [Feature Requests]
Is it possible to allow navigating to a specific tab?

**Comment by kinghat** [comment]
> Is it possible to allow navigating to a specific tab?

what do you mean by "navigate to a specific tab"?

**Reply by cloudbr34k84** [reply]
sorry have a look at these screenshots, see if these make sense. 
![image](https://user-images.githubusercontent.com/58960644/208234843-7e176e47-5a8b-45bf-bb38-b657dea92e5d.png)
![image](https://user-images.githubusercontent.com/58960644/208234849-9aa77a93-f54a-4958-90d5-3b88234cf257.png)

**Reply by kinghat** [reply]
im sorry i forgot to respond to you. im not sure how you would navigate to a card, let alone an element within a card. its an interesting idea and when i get around to making tabs have dynamic properties you might be able to get close. but its a ways off.

**Comment by ncd7** [comment]
I would love this as well but the only difference is my tab card is in another view.

**Comment by shaiger** [comment]
This is the only remaining gap preventing me from replacing lovelace views with the tabbed-card card altogether.
My end goal is to use decluttering templates for "view cards", which will really boost the level of reuse I can reach for my wall panels. 
Bookmarks may serve as a cheap and scalable method than can support multiple tabbed-cards within a single view (this may require naming the card itself).

**Comment by raresdragan** [comment]
@kinghat 

I think it would be amazing to be able to navigate to a specific tab 
Maybe via anchors/ querystring / URL

Here's an example:

I am using the tabbed card to split some of the features into "sections" 
I did split my power related functionalities into tabs like: power / switches / batteries

I have an alert chip for low battery.
When clicking on the low battery alert I should be able to get directly to the opened batteries tab 
(e.g. /lovelace/power/#lowbatterytab)

Can't you add something like anchors based on some tab ids?

https://myhomeassistant:8123/lovelace/power/#tab1
https://myhomeassistant:8123/lovelace/power/#tab2
https://myhomeassistant:8123/lovelace/power/#tab3

What do you think?

<img width="1352" alt="Screenshot 2025-04-19 at 19 51 09" src="https://github.com/user-attachments/assets/6b621bfc-ee21-4fb7-96d8-3f9ef7973f45" />
